migration/data/course_tables.sql

```diff
@@ -498,6 +498,25 @@ CREATE TABLE regrade_discussion (
     deleted BOOLEAN DEFAULT FALSE NOT NULL
 );
 
+--
+-- Name: notifications_type_enum; Type: ENUM; Schema: public; Owner: -
+--
+CREATE TYPE notifications_type AS ENUM ('forum');
+
+--
+-- Name: notifications; Type: TABLE; Schema: public; Owner: -
+--
+CREATE TABLE notifications (
```

Comment:
We might want to add a `Notification` class in the Model to represent this table